### DISCUSSION TITLE
how to fix the issue of failed to execute /tmp/fixup.sh

### DISCUSSION THREAD

**Discussion by diggerdu** [RuriOSS/daijin#51] [General]
![图片](https://github.com/Moe-hacker/daijin/assets/5636045/e65812d6-f2dc-4725-bc62-2a63816e8821)
https://github.com/Moe-hacker/daijin/blob/5e01358bc72ed267ed206404da96d7a455739ee8/src/share/create.sh#L65
From this line of code, it can be inferred that it is caused by ${CONTAINER_DIR} being empty.

**Comment by diggerdu** [comment]
上面的猜测是错的，这个文件存在，到不知道为什么执行不了

**Comment by diggerdu** [comment]
![图片](https://github.com/Moe-hacker/daijin/assets/5636045/f66d8d10-a0bc-4350-8ac4-efc533da88e9)
在termux里单独执行了一下这条命令 发现它访问到的/tmp 是原始环境下面的路径 并没有chroot到rootfs目录 所以找不到/tmp/fixup.sh

![图片](https://github.com/Moe-hacker/daijin/assets/5636045/6c8c1839-d0d0-4ee9-849b-b0e21ad4499e)

**Comment by Moe-hacker** [comment]
> ![图片](https://github.com/Moe-hacker/daijin/assets/5636045/e65812d6-f2dc-4725-bc62-2a63816e8821)
> https://github.com/Moe-hacker/daijin/blob/5e01358bc72ed267ed206404da96d7a455739ee8/src/share/create.sh#L65
> From this line of code, it can be inferred that it is caused by ${CONTAINER_DIR} being empty.

你可以成功启动容器吗？如果可以，将fixup.sh权限设置为777可否将其执行？
